megamek/src/megamek/common/actions/WeaponAttackAction.java

```diff
@@ -1232,6 +1232,10 @@ else if (wtype.getAtClass() == WeaponType.CLASS_LBX_AC) {
 		                || game.getBoard().getHex(te.getPosition()).containsTerrain(Terrains.ROUGH))) {
 		        toHit.addModifier(+1, "urban guerrilla");
 		    }
+		    if (te.getCrew().getOptions().booleanOption("shaky_stick")
```

Comment:
It is in entity isAirborne()
Attacks by flying units are not affected though so the Aero and VTOL checks should stay. In this case though I think a WiGE would also qualify as an airborne unit. Thoughts before I commit the change?